### PR TITLE
GSR and Evidence drop fix

### PR DESCRIPTION
**Pull request by DatBoiWeep**

### CODE REVIEW

File: resources/usa_evidence/cl_evidence.lua [hunk 1]
```diff
@@ -154,7 +154,7 @@ Citizen.CreateThread(function()
 	while true do
 		Citizen.Wait(10)
 		local playerPed = PlayerPedId()
-		if IsPedShooting(playerPed) and WeaponDropsEvidence(GetSelectedPedWeapon(playerPed))  then
+		if IsPedShooting(playerPed) and not WeaponDropsEvidence(GetSelectedPedWeapon(playerPed)) then
```
Comment (minipunch) [inline resources/usa_evidence/cl_evidence.lua:157]:
```suggestion
		if IsPedShooting(playerPed) and not NO_EVIDENCE_WEPS[GetSelectedPedWeapon(playerPed)] then
```

Comment (minipunch) [reply resources/usa_evidence/cl_evidence.lua:157]: and then you can remove the `WeaponDropsEvidence` function all together (but make sure it's also not being used anywhere else first)

**Review by minipunch — CHANGES_REQUESTED** [review]

File: resources/usa_evidence/cl_evidence.lua (hunk 1 above)
Comment (DatBoiWeep) [reply resources/usa_evidence/cl_evidence.lua]: I looked into (Ctrl+Shift+F) the `WeaponDropsEvidence` seems like it's used nowhere else other than here. So she's mint

**Review by minipunch — APPROVED** [review]

**Comment by minipunch** [comment]
👍 nice

**Pull request merged**